stdlib/http/client.pyi

```diff
@@ -101,7 +101,7 @@ class HTTPMessage(email.message.Message):
 
 def parse_headers(fp: io.BufferedIOBase, _class: Callable[[], email.message.Message] = ...) -> HTTPMessage: ...
 
-class HTTPResponse(io.BufferedIOBase, BinaryIO):
+class HTTPResponse(io.BufferedIOBase, BinaryIO):  # type: ignore[misc]  # https://github.com/python/mypy/issues/14002
```

Comment:
We do still need this `type: ignore`, but not because of the mypy issue linked (which is now closed) -- we need it because the definitions of `writelines()` in the two base classes are incompatible. The same goes for all the `type: ignore`s in `io.pyi` and the one in `lzma.pyi`.

Reply:
Updated these comments.